modules/gitops-creating-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-creating-rolloutmanager-custom-resource_{context}"]
+= Creating a RolloutManager custom resource
+
+To manage progressive delivery of deployments by using Argo Rollouts in {gitops-title}, you must create and configure a `RolloutManager` custom resource (CR) in the namespace of your choice. By default, any new `argo-rollouts` instance has permission to manage resources only in the namespace where it is deployed, but you can use Argo Rollouts in multiple namespaces as required.
+
+.Prerequisites
+
+* {gitops-title} 1.9.0 or a newer version is installed in your cluster.
+
+.Procedure 
+
+. Log in to the {product-title} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to create and configure a `RolloutManager` custom resource (CR) from the *Project* drop-down menu.
+
+. Select *OpenShift GitOps Operator* from the installed operators.
+
+. In the *Details* tab, under the *Provided APIs* section, click *Create instance* in the *RolloutManager* pane.
+
+. On the *Create RolloutManager* page, select the *YAML view* and use the default YAML or edit it according to your requirements:
++
+.Example: RolloutManager CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: argo-rollout
+  labels:
+    example: basic
+spec: {}
+----
+
+. Click *Create*.
+
+. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the RolloutManager instance shows as *Phase: Available*.
+
+. In the left navigation pane, verify the creation of the namespace-scoped supporting resources:
++ 
+* Click *Workloads* -> *Deployments* to verify that the `argo-rollouts` deployment is available with the *Status* showing as `1 of 1 pods` running.
+* Click *Workloads* -> *Secrets* to verify that the `argo-rollouts-notification-secret` secret is available.
+* Click *Networking* -> *Services* to verify that the `argo-rollouts-metrics` service is available.
+* Click *User Management* -> *Roles* to verify that the `argo-rollouts` role and `argo-rollouts-aggregate-to-admin`, `argo-rollouts-aggregate-to-edit`, and `argo-rollouts-aggregate-to-view` cluster roles are available.
+* Click *User Management* -> *RoleBindings* to verify that the `argo-rollouts` role binding is available.
```

Comment:
This last step could optionally be converted to a `.Verification` section, if you'd like (per https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs).

If you do, it would need some asterisk-modifications since it would become a single-step procedure (with substeps):

```suggestion
.Verification

* In the left navigation pane, verify the creation of the namespace-scoped supporting resources: 
** Click *Workloads* -> *Deployments* to verify that the `argo-rollouts` deployment is available with the *Status* showing as `1 of 1 pods` running.
** Click *Workloads* -> *Secrets* to verify that the `argo-rollouts-notification-secret` secret is available.
** Click *Networking* -> *Services* to verify that the `argo-rollouts-metrics` service is available.
** Click *User Management* -> *Roles* to verify that the `argo-rollouts` role and `argo-rollouts-aggregate-to-admin`, `argo-rollouts-aggregate-to-edit`, and `argo-rollouts-aggregate-to-view` cluster roles are available.
** Click *User Management* -> *RoleBindings* to verify that the `argo-rollouts` role binding is available.
```

Reply:
Thanks, @adellape for the suggestion. There is a plan to do a deep restructuring of GitOps docs for the upcoming releases. The separation of verification steps will be considered then. So retaining them as is for now.